tests/Persistence/Test_TxPersistence.cpp

```diff
@@ -20,6 +20,7 @@
 #include <thread>
 #include <vector>
 
+#include "depends/libDatabase/LevelDB.h"
```

Comment:
Why does this need to be here? Seems unrelated

Reply:
probably overlooked in merge not anything we wished to change.

Reply:
corrected, we have clion too :-)